system/lib/wasmfs/wasmfs.cpp

```diff
@@ -13,6 +13,14 @@
 #include "special_files.h"
 #include "wasmfs.h"
 
+#ifdef WASMFS_CASE_INSENSITIVE
+#include <emscripten/wasmfs.h>
+
+static backend_t make_mem_backend(void* arg) {
```

Comment:
Please add a comment here mentioning what this is used for.

Reply:
Removed that and used a lambda instead.